src/frontend/src/binder/expr/function.rs

```diff
@@ -186,6 +186,29 @@ impl Binder {
                 };
                 return Ok(ExprImpl::literal_varchar(v));
             }
+            "rw_version" => {
+                if !inputs.is_empty() {
+                    return Err(ErrorCode::ExprError(
+                        "rw_version() does not accept any arguments".into(),
+                    )
+                    .into());
+                }
+                let version: String;
+                #[cfg(debug_assertions)]
+                {
+                    let output = std::process::Command::new("git")
+                        .args(["rev-parse", "HEAD"])
+                        .output()
+                        .unwrap();
```

Comment:
This probes the current working directory of frontend node at runtime and may not be accurate in the following cases:
* Build and start an instance
* Switch working directory to another commit, forgetting killing the instance
* `select rw_version();`

@BugenZhao Any workarounds for the build time issue in https://github.com/risingwavelabs/risingwave/pull/2636#issuecomment-1129775437 ?

Reply:
Do we really need a debug version?

Reply:
+1 
seems useless